driver/driver_test.go

```diff
@@ -30,3 +32,49 @@ func TestDebugDriver_Run(t *testing.T) {
 	}
 	is.NoError(d.Run(op))
 }
+
+func TestOperation_Unmarshall(t *testing.T) {
+	var op Operation
+	is := assert.New(t)
+	bytes, err := ioutil.ReadFile("../testdata/operations/valid-operation.json")
+	is.NoError(err, "Error reading from testdata/operations/valid-operation.json")
+	is.NoError(json.Unmarshal(bytes, &op), "Error unmarshalling operation")
+	is.NotNil(op, "Expected Operation not to be nil")
```

Comment:
Could you test with a `DeepEqual` instead? So you are sure the unmarshalled struct is valid.